package.json

```diff
@@ -11,8 +11,9 @@
   "scripts": {
     "lint": "eslint index.js test",
     "unittest": "mocha test/*.spec.js",
-    "cover": "istanbul cover _mocha -- -R spec test/*.spec.js; istanbul check-coverage",
-    "test": "npm run lint && npm run cover"
+    "cover": "nyc mocha test/*.spec.js; nyc check-coverage --lines 100 --statements 100 --branches 100 --functions 80",
```

Comment:
The functions coverage has never been 100% so this is ok. The istanbul.yml was never used.